### PR TITLE
XW-2076 | preparing Footer for rebranding

### PR DESCRIPTION
**Pull request by Warkot**
* https://wikia-inc.atlassian.net/browse/XW-2076
* https://github.com/Wikia/app/pull/11398

Footer has to be ready to properly render these changes: https://github.com/Wikia/app/pull/11396.

@Wikia/x-wing

### CODE REVIEW

**Comment by hakubo** [comment]
Should we update reference page?

File: components/_global-footer.scss [hunk 1]
```diff
@@ -63,6 +63,7 @@
 		display: flex;
 		justify-content: center;
 		margin: 0;
+		padding: 30px 0 25px;
```
Comment (Warkot) [inline components/_global-footer.scss]: Does it look how it should with the current logos?

**Comment by Warkot** [comment]
Yes, we should update the reference page. We should also support the case where there's no `wds-is-en` or `wds-is-international` class added. I'll cover these.

File: components/_global-footer.scss [hunk 2]
```diff
@@ -112,6 +113,11 @@
 		text-transform: uppercase;
 	}
 
+	&__header-logo {
+		height: 45px;
+		width: 185px;
```
Comment (Warkot) [inline components/_global-footer.scss]: Why 185 x 45? SVG has `width="184" height="44"`.

Comment (hakubo) [inline components/_global-footer.scss]: I like round numbers :)

File: assets/company/logo-fandom-powered-by-wikia.svg [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <svg width="254" height="26" viewBox="0 0 254 26" xmlns="http://www.w3.org/2000/svg">
   <defs>
-    <linearGradient x1="0%" y1="100%" x2="63.848%" y2="32.54%" id="logo-fandom-powered-by-wikia-gradient">
+    <linearGradient x1="0%" y1="100%" x2="63.848%" y2="32.54%" id="a">
```
Comment (rogatty) [inline assets/company/logo-fandom-powered-by-wikia.svg]: This is not a good change, I think we should manually revert / fix all those ids to be unique.

Comment (Warkot) [inline assets/company/logo-fandom-powered-by-wikia.svg]: That was probably caused by `svgo` run without config. I'll revert that.

**Comment by kvas-damian** [comment]
In this PR we changes logo spacing for international version - see VRT. Was it intended?

File: components/_global-footer.scss (hunk 2 above)
Comment (Warkot) [inline components/_global-footer.scss]: They're perfectly rounded to full pixel. ;)

File: assets/company/logo-fandom-powered-by-wikia.svg (hunk 3 above)
Comment (Warkot) [inline assets/company/logo-fandom-powered-by-wikia.svg]: Fixed that. However, the new SVG `logo-fandom-powered-by-wikia-two-lines.svg` had just generic IDs like `path-1`. The SVGs merged from `master` also have generic IDs like `a` (they're just reformated).

**Comment by Warkot** [comment]
@hakubo could you answer @kvas-damian's comment https://github.com/Wikia/design-system/pull/68#issuecomment-248563900?

**Comment by hakubo** [comment]
@kvas-damian not intended. but this way we minimise the work needed to be done on Oct 4th. 
as we won't have to add any classes to it.

**Comment by hakubo** [comment]
confirmed with Derek, it's fine to change these paddings

**Comment by kvas-damian** [comment]
Please update VRT!

**Pull request merged**